packages/table/test/columnTests.tsx

```diff
@@ -75,13 +76,16 @@ describe("Column", () => {
         expectCellLoading(columnHeaders[1], CellType.COLUMN_HEADER);
         expectCellLoading(columnHeaders[2], CellType.COLUMN_HEADER, false);
 
-        const col0cells = table.element.queryAll(`.${Classes.columnCellIndexClass(0)}`);
+        const col0CellsClass = classNames(Classes.columnCellIndexClass(0), Classes.TABLE_CELL);
+        const col0cells = table.element.queryAll(col0CellsClass);
```

Comment:
We should probably do a length check on the number of cells, in this test, then. Whoops, though, good catch. Will fix and add length check.